pages/checkout.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Layout from "../components/layout/Layout";
+
+const Checkout = () => {
+  return (
+    <Layout title={"Checkout"} hasFooter={true}>
```

Comment:
You can use the shorthand for boolean react props here:
```suggestion
    <Layout title="Checkout" hasFooter>
```

If your layout doesn't need a footer you can just get rid of the property for that case which for react means that the value for that prop is `undefined` which is [falsy](https://developer.mozilla.org/de/docs/Glossary/Falsy) enough for most cases. 

> also note the `=""` syntax for string props